cmd/statusd/utils.go

```diff
@@ -1397,3 +1394,39 @@ func testValidateNodeConfig(t *testing.T, config string, fn func(common.APIDetai
 
 	fn(resp)
 }
+
+// ensureNodeSync waits until synchronzation is done and more safe
+// than only waiting a fixed amount of time.
+func ensureNodeSync(t *testing.T, nodeManager common.NodeManager) {
+	start := time.Now()
+	wait := 5 * time.Second
+	les, err := nodeManager.LightEthereumService()
+	if err != nil {
+		t.Error(err)
+	}
+	if les == nil {
+		t.Errorf("LightEthereumService is nil")
+	}
+
+	for {
+		// Some time needed even initially.
+		time.Sleep(wait)
```

Comment:
If the blockchain is already synced, we would waste 5 seconds before each test. Can't we skip this for the initial check?

Reply:
Will move.